docs/js/files/modules_ListView.js.html

```diff
@@ -287,6 +295,11 @@ <h1 class="file-heading">File: modules/ListView.js</h1>
     __itemContext: null,
     __renderWait: 0,
     __delayedRender: null,
+    /**
+     * @property __delayedRenderTimeout
+     * @private
+     * @type {Number}
```

Comment:
as far as I know Integer is not a javascript type, there is just Number.  And yuidoc knows about built-in types so it would link {Number} correctly, but {Integer} not correctly.